### DISCUSSION TITLE
Why is Ruby required ?

### DISCUSSION THREAD

**Discussion by VisionovaDev** [tabler/tabler#1898] [General]
In the environment setup instructions there is Ruby as a requirement.
A a PHP backend developer who don't know Ruby I am asking to myself if tabler can be used without install Ruby.

**Comment by dheineman** [comment]
Ruby is required if you want to run the tabler (documentation) project itself, for example when you want to contribute to tabler.

If you simply want to use tabler in your project you can install it using npm
https://github.com/tabler/tabler?tab=readme-ov-file#installation
```
npm install --save @tabler/core
```

Or include the CDN files in your HTML
https://github.com/tabler/tabler?tab=readme-ov-file#cdn-support
```
<script src="https://cdn.jsdelivr.net/npm/@tabler/core@latest/dist/js/tabler.min.js"></script>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@tabler/core@latest/dist/css/tabler.min.css">
````
